packs/feats/derring-do.json

```diff
@@ -24,7 +24,17 @@
             "remaster": false,
             "title": "Pathfinder Advanced Player's Guide"
         },
-        "rules": [],
+        "rules": [
+            {
+                "keep": "higher",
+                "key": "RollTwice",
+                "predicate": [
+                    "panache",
```

Comment:
```suggestion
                    "self:effect:panache",
```

Reply:
I thought it would've been fine since panache's effect gives that roll option, but it's best to be safe